### DISCUSSION TITLE
Not able to use github action: "specifying a destination required for non-interactive publish"

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#2952] [Q&A] [answered]
Hi,
I'm triying to use for the first time a github action to publish a quarto website (under costruction).

[This](https://github.com/ondata/supporto-monitoraggio-pnrr) is the repo.

I have [this error](https://github.com/ondata/supporto-monitoraggio-pnrr/actions/runs/3289320467/jobs/5420706360):

```
Run quarto-dev/quarto-actions/publish@v2
Run git config --global user.email "quarto-github-actions-publish@example.com"
ERROR: No _publish.yml file available (_publish.yml specifying a destination required for non-interactive publish)
Error: Process completed with exit code 1.
```

What's wrong in my configuration?

Thank you

**Comment by aborruso** [comment, marked as answer]
I didn't create the gh-pages branch.
As soon as I did it it worked.

I will set the repo as private, as it is under construction for now

**Reply by benlindsay** [reply]
Thanks! This fixed it for me too!
